xdsl/ir/core.py

```diff
@@ -1615,12 +1615,23 @@ def split_before(
     def get_operation_index(self, op: Operation) -> int:
         """Get the operation position in a block."""
         if op.parent is not self:
-            raise Exception("Operation is not a children of the block.")
+            raise Exception("Operation is not a child of the block.")
         for idx, block_op in enumerate(self.ops):
             if block_op is op:
                 return idx
         assert False, "Unexpected xdsl error"
 
+    def get_operation_by_index(self, idx: int) -> Operation:
```

Comment:
nit:
```suggestion
    def get_operation_at_index(self, index: int) -> Operation:
```
I can't formulate it with words but for some reason at index seems more natural than by index

Reply:
yeah, when in a hurry, prepositions are not my strong suit.